src/scripts/num.js

```diff
@@ -170,5 +201,7 @@ export default {
   percent,
   percentInt,
   prettyDecimals,
-  roundObjectPercentages
+  roundObjectPercentages,
+  bigFigure,

```

Comment:
why `figure` as opposed to `bigNumber` or `truncateBigNumber`?

Reply:
I chose `bigFigure` to distinguish it from the `BigNumber` lib. It could also be called `truncatedBigNumber` but then the naming of the selector filter would also need to change. I think `truncatedBigNumberOrShortDecimals` can already hurt the eyes

Reply:
`truncatedBigNumberOrShortDecimals`

it's odd to me to have an `Or` in this. not really sure when to use it or what it will do.

Reply:
How would you call it then?

Reply:
its ok for now.